src/context/like-context.js

```diff
@@ -21,11 +22,18 @@ const LikedProvider = ({ children }) => {
         );
         if (response.status === 201) {
           setLikedList(response.data.likes);
+          Toast({ type: "success", message: "Added to Liked Videos" });
         }
       } catch (error) {
         console.log("error", error);
-        navigate("/login");
+        Toast({
```

Comment:
So we don't need to pass the config of type. We can directly do this.
![image](https://user-images.githubusercontent.com/67600170/179602335-7e84fdc1-1e50-4fcb-89a9-3acf6bd257a5.png)